app/routes/account_.$accountId.$appId.update/route.tsx

```diff
@@ -53,13 +54,16 @@ export const loader: LoaderFunction = async ({ request, params }) => {
       return redirect(`/account/${params.accountId}`)
     }
 
-    const isUserMember =
-      getUserRole(
-        getUserPortalAppResponse.getUserPortalApp.portalAppUsers,
-        user.user.portalUserID,
-      ) === "MEMBER"
+    const currentAccount = getUserAccountsResponse?.getUserAccounts?.find(
```

Comment:
im not following why we would need to do this. don't we already stop all users not associated to the account at the account root.

Reply:
also, wouldnt it just make sense to use the accountid.update route instead of having the logic twice?

Reply:
This is for the app update. If a member user manually goes to `$appId/update`.. we redirect them back.